htdocs/modules/profile/class/profile.php

```diff
@@ -349,7 +349,9 @@ public function search(CriteriaElement $criteria, $searchvars = array(), $groups
         if ((!empty($limit) && $count >= $limit) || !empty($start)) {
             $sql_count = 'SELECT COUNT(*)' . $sql_from . $sql_clause;
             $result    = $this->db->query($sql_count);
-            list($count) = $this->db->fetchRow($result);
+            if (!$this->db->isResultSet($result)) {
+                \trigger_error("Query Failed! SQL: $sql_count- Error: " . $this->db->error(), E_USER_ERROR);
+            }
         }
```

Comment:
Don't we still need the ```list($count) = $this->db->fetchRow($result);``` after we are sure we got a real result set?

Reply:
Ouch! Yes, of course! It's now fixed!